core/src/main/java/org/apache/accumulo/core/util/SimpleThreadPool.java

```diff
@@ -28,15 +28,32 @@
  */
 public class SimpleThreadPool extends ThreadPoolExecutor {
 
-  public SimpleThreadPool(int max, final String name) {
-    super(max, max, 4L, TimeUnit.SECONDS, new LinkedBlockingQueue<>(),
+  // the number of seconds before we allow a thread to terminate with non-use.
+  public static final long DEFAULT_TIMEOUT_MILLISECS = 180000L;
+
+  public SimpleThreadPool(int coreAndMax, final String name) {
+    this(coreAndMax, DEFAULT_TIMEOUT_MILLISECS, name);
+  }
+
+  public SimpleThreadPool(int coreAndMax, long threadTimeOut, final String name) {
+    super(coreAndMax, coreAndMax, threadTimeOut, TimeUnit.MILLISECONDS, new LinkedBlockingQueue<>(),
```

Comment:
Would it work do only the following in the constructor?

```suggestion
    this(coreAndMax, threadTimeOut, name, new LinkedBlockingQueue<>());
```

Reply:
That would work and would be cleaner.  I could also them remove the redundant lines after.  I will add this change.

Reply:
done